examples/chart/teleport-demo/docker/build-all.sh

```diff
@@ -1,15 +1,25 @@
-#!/usr/bin/env bash
+#!/bin/bash
 set -e
+ONE=$1
+TWO=$2
 VERSION=3.2.0
 if [[ "$1" != "" ]]; then
     VERSION=$1
     shift
 fi
-set -e
+GCPROJECT=kubeadm-167321
```

Comment:
Put this at the top of the file (just after `VERSION`) so it's before any comparison logic. It's easier to find variables when they're all set in one place.